### PR TITLE
 Add example for grant execute external script on external language

### PR DESCRIPTION
**Pull request by twright-msft**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@twright-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/t-sql/statements/create-external-language-transact-sql.md [hunk 1]
```diff
@@ -152,6 +152,15 @@ FROM
 (CONTENT = N'<path-to-tar.gz>', FILE_NAME = 'javaextension.so', PLATFORM = LINUX);
 GO
 ```
+### C. Grant permissions to execute external script
+
+The following example garants the mylogin principal access to execute scripts using the "Java" external language.
```
Comment (dphansen) [inline docs/t-sql/statements/create-external-language-transact-sql.md]: Please accept the following changes:

```suggestion
The following example grants the **mylogin** principal access to execute scripts using the **Java** external language.
```

**Comment by dphansen** [comment]
@twright-msft I've reviewed the PR. Please commit the suggested change and I will get it signed off. Thanks.

**Comment by PRMerger14** [comment]
@twright-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dphansen** [comment]
#sign-off

**Comment by PRMerger15** [comment]
@dphansen: I'm sorry - only the author of this article, @nelgson, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**